http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -305,6 +306,7 @@ case class ClusterInitValues private (googleProject: String,
 
 object ClusterInitValues {
   val serviceAccountCredentialsFilename = "service-account-credentials.json"
+  val customEnvVarFilename = "custom-env-vars.env"
```

Comment:
In `jupyter-docker-compose` it expects a file named `custom_env_vars.env` -- this should probably be consistent.